sources/api/migration/migrator/src/main.rs

```diff
@@ -185,7 +185,11 @@ pub(crate) fn run(args: &Args) -> Result<()> {
 /// Generates a random ID, affectionately known as a 'rando', that can be used to avoid timing
 /// issues and identify unique migration attempts.
 fn rando() -> String {
-    thread_rng().sample_iter(&Alphanumeric).take(16).collect()
+    thread_rng()
+        .sample_iter(&Alphanumeric)
+        .take(16)
+        .map(|c| char::from(c))
```

Comment:
Just curious: was syntax ` .map(char::from)`
    not working.  Reference from their doc: https://docs.rs/rand/0.8.0/rand/distributions/trait.Distribution.html#example

Reply:
You are correct, I know that clippy would have compelled me to do so had I used it.